data_fixtures/migrations/0003_unit_page_copy_updates.py

```diff
@@ -0,0 +1,95 @@
+# Generated by Django 4.2.14 on 2024-07-16 17:30
+
+from django.db import migrations
+
+fixtures = [
+    {
+        "name": "mitpe",
+        "offeror_configuration": {
+            "value_prop": (
+                "MIT Professional Education is a leader in technology and "
+                "engineering education for working professionals pursuing "
+                "career advancement, and organizations seeking to meet modern-day "
+                "challenges by expanding the knowledge and skills of their employees. "
+                "Courses are delivered in a range of formats—in-person (on-campus "
```

Comment:
Should there be spaces around the dashes so it's `formats — in-person` instead of `formats—in-person`?

Alternatively so there are fewer different kinds of dashes `Courses are delivered in a range of formats to meet the needs of today's learners: in-person (on-campus and live virtual), online, and through hybrid approaches`

Reply:
Its hard to discern in plain text but those are em dashes. once you load the fixture it displays as expected. 
<img width="293" alt="Screenshot 2024-07-17 at 11 59 23 AM" src="https://github.com/user-attachments/assets/a921b892-25a7-49fe-bf38-afe1bd2ad85c">